data/map.txt

```diff
@@ -26492,7 +26492,7 @@ planet "Buccaneer Bay"
 	landscape land/beach0
 	description `Pirates who have struck it rich dream of retiring on Buccaneer Bay. It is a warm ocean planet, with many places reminiscent of the islands in the ancient Caribbean where humanity's first pirate fleets sailed.`
 	description `	Many of the inhabitants are in fact not pirates, but their descendants, and these children and grand-children of anarchist outlaws have built a system of regional government that oversees hospitals, schools, and other public services, all created without funding from the Republic or taxes paid to it. The planet even sports its own shipyard, although many of the ships built here are, not surprisingly, based on designs pirated from the Syndicate and Republic shipyards.`
-	spaceport `This is the sort of spaceport you picture in your head when you think of pirates. Half the storefronts are bars, and rum seems to be on the menu everywhere. It is almost like the stories of ancient Earth, except that the eye patches are cybernetic implants and the peg legs are motorized prosthetics. Swarthy, dangerous-looking women glance at you as you pass by, and scantily clad men crowd around the bars, which is what you seem to remember the stories said it was like on Earth, too, although there it may have been the other way around.`
+	spaceport `This is the sort of spaceport you picture in your head when you think of pirates. Half the storefronts are bars, and rum seems to be on the menu everywhere. It is almost like the stories of ancient Earth, except that the eye patches are cybernetic implants and the peg legs are motorized prosthesis. Swarthy, dangerous-looking women glance at you as you pass by, and scantily clad men crowd around the bars, which is what you seem to remember the stories said it was like on Earth, too, although there it may have been the other way around.`
```

Comment:
"Prosthetic" is the language used elsewhere in the game, so if we're attempting to change one word for another as was done with "liquify" and "miniscule," we're missing some. Even without that though, this is actually wrong now; you've replaced the plural "prosthetics" with the singular "prosthesis," when it should be the plural "prostheses." 

I prefer "prosthetic" to "prosthesis" though, so this particular change should actually just be reverted.

Reply:
You're right about the plural being prostheses, my mistake.
However "prosthetic" is actually an adjective. A prosthetic leg is a prosthesis. Comparing those words on Google Ngram Viewer isn't relevant since they have always been used in parallel.
Like the other words I replaced, it looks more like a mistake that is so common that many people have accepted it.

I have checked the game files, and there is no other occurrence of the word "prosthetic" being used as a noun (nor any other "miniscule", "liquify" or "unmistakingly"), while "prosthesis" already appears in a Remnant mission.

Reply:
fixed in eb09c7f72005d380d3cf0688c18f97e7553bafe4